run.sh

```diff
@@ -15,7 +15,7 @@ cd carapace-server-${CARAPACE_V}
 timeout 22 sh -c 'until nc -z $0 $1; do sleep 1; done' localhost 8001
 
 # apply dynamic configuration
-curl -X POST --data-binary @conf/server.dynamic.properties http://localhost:8001/api/config/apply --user admin:admin -H "Content-Type: text/plain"
+curl -X POST --data-binary @conf/server.dynamic.properties http://$(hostname):8001/api/config/apply --user admin:admin -H "Content-Type: text/plain"
```

Comment:
leave it as "localhost", it is the safest value by default.
This script is meant only for developers and testers, not for production.

Please also add a license header to this file